src/screens/doc/DocIconsScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../../components';
+import { colors, Icon, normalize } from '../../config';
+
+type TIconItem = {
+  iconName: keyof typeof Icon;
+};
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+
+  return (
+    <View style={styles.iconContainer}>
+      <SelectedIcon size={normalize(35)} color={colors.secondary} />
```

Comment:
what about `<SelectedIcon />` to have it as short as possible? there is no need for special adjustment, right? that way we have the icons 1:1 from the configs:

![Simulator Screen Shot - iPhone 14 - 2023-09-15 at 16 20 03](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/a0d3744f-05d2-4496-a7db-a3e88041c245)

Reply:
I figured some icons have a better visibility with a darker color on the white background, like e.g. EmptySection in your screenshot above. 
Tell me what you think about this idea, here is a comparison screenshot to yours above: 

<img src=https://github.com/smart-village-solutions/smart-village-app-app/assets/105894697/cd1232ca-d406-47e0-b0b8-b6ec586190ca width="300">

Reply:
@donni106

Reply:
it's not about best visibility in first place, but we want to see icons. to understand when looking on them, i suggest not to modify them. when modifying for documentation we cannot draw conclusions to the rest of the code.
color: mostly shown as primary in the app -> should be primary here
size: mostly shown in default size configured in the app -> should be the same here as much as possible
the empty sections icon is special in size, yes. we do not need a properly visible size in the documentation, as it is the priority to see the icon and to know which icons are available.